cache/core.go

```diff
@@ -4,51 +4,44 @@ import (
 	"database/sql"
 	"time"
 
-	log "github.com/Sirupsen/logrus"
-
 	"github.com/Bnei-Baruch/archive-backend/utils"
 )
 
-type Refreshable interface {
```

Comment:
This should be removed.
You can inject SearchStatsCacheImpl with dependencies (args to refresh) in other ways (constructor, setters, etc...)

Reply:
changed